tests/test_transform.py

```diff
@@ -95,6 +95,11 @@ def test_search_space_transform_numerical(
     expected_low = distribution.low  # type: ignore
     expected_high = distribution.high  # type: ignore
 
+    if isinstance(distribution, UniformDistribution) or isinstance(
+        distribution, LogUniformDistribution
+    ):
```

Comment:
Mostly a cosmetic but you can shorten this.
```suggestion
    if isinstance(distribution, (UniformDistribution, LogUniformDistribution)):
```